apps/web/src/utils/contractHelpers.ts

```diff
@@ -446,3 +454,39 @@ export const getFixedStakingContract = (signer?: WalletClient, chainId?: number)
     chainId,
   })
 }
+
+export const getVeCakeContract = (signer?: WalletClient, chainId?: number) => {
+  return getContract({
+    abi: veCakeABI,
+    address: getVeCakeAddress(chainId) ?? getVeCakeAddress(ChainId.BSC_TESTNET),
```

Comment:
Add a note here in case we need to default to use bsc contract addresses before launch